src/memberdef.cpp

```diff
@@ -2470,7 +2470,19 @@ void MemberDefImpl::writeDeclaration(OutputList &ol,
         ol.pushGeneratorState();
         ol.disableAllBut(OutputGenerator::Html);
         ol.docify(" ");
-        ol.startTextLink(getOutputFileBase(),anchor());
+        MemberDefMutable *annMemb = NULL;
+        if (!isAnonymous()) // hide anonymous stuff
```

Comment:
shouldn't this be `if (isAnonymous())`?

Reply:
I was wondering as well, though in the "write name" there was also `!isAnonymous()`, see line 2267.